feconf.py

```diff
@@ -213,14 +213,23 @@
 # for recently published explorations in /library/recently_published page.
 RECENTLY_PUBLISHED_QUERY_LIMIT_FULL_PAGE = 20
 
+# The minimum version of the Android feedback report info blob schema.
+MIN_ANDROID_REPORT_SCHEMA_VERSION = 1
```

Comment:
Can you use MINIMUM_* instead for all three of these? We try not to abbreviate.

Reply:
Done